openslides/mediafiles/serializers.py

```diff
@@ -1,14 +1,38 @@
-from openslides.utils.rest_api import ModelSerializer, SerializerMethodField
+import mimetypes
 
+from django.db import models as dbmodels
+from rest_framework.fields import FileField
+
+from ..utils.rest_api import ModelSerializer, SerializerMethodField
 from .models import Mediafile
 
 
+class AngularCompatibleFileField(FileField):
+
+    def to_internal_value(self, data):
+        if data == '':
+            return None
+        return super(AngularCompatibleFileField, self).to_internal_value(data)
+
+    def to_representation(self, value):
+        if not value:
+            return None
+        return {
+            'name': value.name,
+            'type': mimetypes.guess_type(value.path)[0]
```

Comment:
What do you mean? This Field costomizes the serialization of FileFields. I thought, this would be a nice, reusable way to implement a serialization which the angular upload module likes.